DigitalLearningSolutions.Web/Controllers/TrackingSystem/Centre/Reports/ReportsController.cs

```diff
@@ -173,6 +173,47 @@ ReportInterval reportInterval
             );
         }
 
+        [HttpGet]
+        [Route("DownloadEvaluationSummaries")]
+        public IActionResult DownloadEvaluationSummaries(
+            int? jobGroupId,
+            int? courseCategoryId,
+            int? customisationId,
+            string startDate,
+            string endDate,
+            ReportInterval reportInterval
+            )
+        {
+            var centreId = User.GetCentreId();
+            var adminId = User.GetAdminId()!.Value;
+            var adminUser = userDataService.GetAdminUserById(adminId)!;
+
+            var dateRange =
+                activityService.GetValidatedUsageStatsDateRange(startDate, endDate, centreId);
+
+            if (dateRange == null)
+            {
+                return new NotFoundResult();
+            }
+
+            var filterData = new ActivityFilterData(
+                dateRange.Value.startDate,
+                dateRange.Value.endDate,
+                jobGroupId,
+                adminUser.CategoryId == 0 ? courseCategoryId : adminUser.CategoryId,
+                customisationId,
+                customisationId.HasValue ? CourseFilterType.Course : CourseFilterType.CourseCategory,
+                reportInterval
+            );
+
+            var content = evaluationSummaryService.GetEvaluationSummaryFileForCentre(centreId, filterData);
+            return File(
+                content,
+                FileHelper.ExcelContentType,
+                $"DLS Evaluation Stats {DateTime.Today:yyyy-MM-dd}.xlsx"

```

Comment:
No, this is the same date format we use for other downloads, and the best date format for naming files anyway (so that they show in the correct order in file explorer)